src/components/buttons/_buttons-mixins.scss

```diff
@@ -32,85 +25,213 @@ $buttonActiveScale: scale(0.985);
   white-space: nowrap;
   font-weight: $fontWeightBold;
   text-transform: uppercase;
-  box-shadow: $componentShadow;
-  transition-property: box-shadow, transform, opacity;
+  box-shadow: none;
+  transition-property: background-color, border-color;
   transition-duration: 0.3s;
+  transition-timing-function: ease-in-out;
 
-  // hack for proper text centering on Safari related to inline-flex issue with <button> tag
-  &::before,
-  &::after {
-    content: '';
-    flex: 1 0 auto;
-  }
-
-  &:hover,
-  &:focus {
-    box-shadow: $elevatedComponentShadow;
+  &__container {
+    display: flex;
+    align-items: center;
+    justify-content: center;
   }
 
-  &:active {
-    box-shadow: $componentShadow;
-    transform: $buttonActiveScale;
+  &--full-width {
+    width: 100%;
   }
 
-  &:active:focus {
-    box-shadow: $buttonActiveBoxShadow;
+  &:focus {
+    outline: none;
```

Comment:
not for now, but actually would be good to have some outline here in the future.